### DISCUSSION TITLE
phot_fmt for Model.plot_lc(), Event.plot(), etc.

### DISCUSSION THREAD

**Discussion by jenniferyee** [rpoleski/MulensModel#50] [Ideas]
I noticed today that while there is a phot_fmt keyword for MulensData.plot(), the corresponding keyword does not exist for Models (and so also does not exist for Event methods that plot models). I think we should add this keyword (I needed it today).

Also, it appears that the yaxis is still labeled 'magnitude' and is still flipped for Event.plot_data(phot_fmt='flux').

**Comment by rpoleski** [comment]
You want to plot fluxes instead of magnitudes? That could be added.
